### DISCUSSION TITLE
LImit getTableQuery() records in Widget

### DISCUSSION THREAD

**Discussion by korematter** [filamentphp/filament#6318] [Help] [answered]
Hi, 

I'm trying to limit the getTableQuery() records  to 1 or any other number in a Widget in a Dashboard but it seems the Builder query always returns a "limit 11 offset 0".  Is there a way to override this?  Thank you.

```
class UpcomingCompetitions extends BaseWidget
{

    protected int | string | array $columnSpan = 'full';

    protected function getTableQuery(): Builder
    {
        return Competition::where('status_id', 20)->take(1);
    }
}
```

![image](https://user-images.githubusercontent.com/58536534/234452005-e6f45f0d-321e-4b93-a37c-2a6bab7bb65f.png)

**Comment by ManojKiranA** [comment, marked as answer]
You can use `take` or `limit` together with `isTablePaginationEnabled`


```
protected function getTableQuery(): Builder
{
  return Competition::where('status_id', 20)->take(5);
}
```
```
protected function isTablePaginationEnabled(): bool
{
   return false;
}
```

**Reply by korematter** [reply]
Ah, thank you! That worked. Didn't realize that query was a result of pagination. Makes sense now. Thanks again!
